Model/Indexer/AsyncEventSubscriber.php

```diff
@@ -78,24 +81,16 @@ public function __construct(
      * @inheritDoc
      */
     public function executeFull()
-    {
-        $this->execute([]);
-    }
-
-    /**
-     * @inheritDoc
-     */
-    public function execute($ids)
     {
         foreach ($this->dimensionProvider->getIterator() as $dimension) {
-            $this->executeByDimensions($dimension, new ArrayIterator($ids));
+            $this->executeByDimensions($dimension);
         }
     }
 
     /**
      * @inheritDoc
      */
-    public function executeByDimensions(array $dimensions, Traversable $entityIds)
+    public function executeByDimensions(array $dimensions, Traversable $entityIds = null)
```

Comment:
Would `?Traversable` be a better argument type here?

Reply:
They are :smile: I'll update them!